.circleci/config.yml

```
@@ -119,3 +116,14 @@ jobs:
       - store_artifacts:
           path: /root/project/smoketest/cypress/videos
           destination: videos
+
+  pr-success:
```

YAMLlint (1.37.1)

[error] duplication of key "pr-success" in mapping

(key-duplicates)